### PR TITLE
feat: add datetime options to xAxis, yAxis

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

image

```js
const options = {
  xAxis: {
    date: { format: 'yyyy-MM'}
  },
  yAxis: {
    date: { format: 'yyyy-MM'}
  }
}
```

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/store/axes.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ import {
   LineTypeXAxisOptions,
   BoxSeriesOptions,
   RangeDataType,
-  Rect,
+  Rect, DateOption,
```
Comment (lja1018) [inline src/store/axes.ts:35]: prettier 적용이 안됬네요

**Review by lja1018 — APPROVED** [review]
[09/22] 리뷰완료
고생하셨습니다!

File: types/options.d.ts [hunk 2]
```diff
@@ -172,13 +172,11 @@ type YAxisOptions = BaseAxisOptions & {
 
 type BothSidesYAxisOptions = YAxisOptions | YAxisOptions[];
 
+type DateOption = boolean | { format: string };
```
Comment (jungeun-cho) [inline types/options.d.ts:175]: d.ts파일은 명시적으로 export 해주지 않아도 외부 파일에서 잡혀서 사용할 수 있나요?;

**Review by jungeun-cho — APPROVED** [review]
[09/23] 리뷰완료합니다. 수고하셨어요~

File: types/options.d.ts (hunk 2 above)
Comment (jung-han) [reply types/options.d.ts:175]: 넵 그렇다고 하네요

**Pull request merged**